docs/source/portal-molecule.rst

```diff
@@ -23,21 +23,68 @@ Molecules can be created via NumPy, Psi4-like molecule strings, and QCSchema JSO
+
+Optional Keys
++++++++++++++
+
+   +-----------------------------------------------+--------------------------------------------------------------------------------------------------------------------------+-----------------------------+
+   | Key Name                                      | Description                                                                                                              | Field Type                  |
+   +===============================================+==========================================================================================================================+=============================+
+   | fragment_charges                              | (nfr, ) list of charges associated with each fragment tuple.                                                             | array[number]               |
+   +-----------------------------------------------+--------------------------------------------------------------------------------------------------------------------------+-----------------------------+
+   | fragment_multiplicities                       | (nfr, ) list of multiplicities associated with each fragment tuple.                                                      | array[number]               |
+   +-----------------------------------------------+--------------------------------------------------------------------------------------------------------------------------+-----------------------------+
+   | real                                          | (nat, ) list describing if atoms are real (T) or ghost (F).                                                              | array[boolean]              |
+   +-----------------------------------------------+--------------------------------------------------------------------------------------------------------------------------+-----------------------------+
+   | comment                                       | Any additional comment one would attach to the molecule.                                                                 | string                      |
+   +-----------------------------------------------+--------------------------------------------------------------------------------------------------------------------------+-----------------------------+
+   | connectivity                                  | (nat, ) list describing bonds within a molecule. Each element is a (atom1, atom2, order) tuple.                          | array[array]                |
+   +-----------------------------------------------+--------------------------------------------------------------------------------------------------------------------------+-----------------------------+
+   | provenance                                    | #/definitions/provenance                                                                                                 | object                      |
```

Comment:
Not sure if this was suppose to link somewhere like a `:term:` directive, but it looks odd. If its placeholder, thats also fine for now